packages/functional-tests/tests/subscription-tests/subscription.spec.ts

```diff
@@ -94,147 +94,141 @@ test.describe('severity-2 #smoke', () => {
   });
 
   test.describe('Flow, acquisition and new user checkout funnel metrics', () => {
-    let metricsObserver: MetricsObserver;
-
-    test.beforeEach(({ pages: { subscribe } }) => {
+    test.beforeEach(() => {
       test.slow();
-      metricsObserver = new MetricsObserver(subscribe);
-      metricsObserver.startTracking();
     });
 
-    test.describe('severity-2', () => {
-      test('Metrics disabled: existing user checkout URL to not have flow params', async ({
-        pages: { settings, relier, page },
-      }, { project }) => {
-        test.skip(
-          project.name === 'production',
-          'test plan not yet available in prod'
-        );
-        // Go to settings page and verify the Data Collection toggle switch is visible
-        await settings.goto();
-        expect(await settings.dataCollection.isToggleSwitch()).toBe(true);
-
-        // Toggle swtich, verify the alert bar appears and its status
-        await settings.dataCollection.toggleShareData('off');
-        expect(await settings.alertBarText()).toContain('Opt out successful.');
-        expect(await settings.dataCollection.getToggleStatus()).toBe('false');
-
-        // Subscribe to plan and verify URL does not include flow parameter
-        await relier.goto();
-        await relier.clickSubscribe6Month();
-        expect(page.url()).not.toContain('&flow_begin_time=');
-      });
+    test('Metrics disabled: existing user checkout URL to not have flow params', async ({
+      pages: { settings, relier, page },
+    }, { project }) => {
+      test.skip(
+        project.name === 'production',
+        'test plan not yet available in prod'
+      );
+      // Go to settings page and verify the Data Collection toggle switch is visible
+      await settings.goto();
+      expect(await settings.dataCollection.isToggleSwitch()).toBe(true);
+
+      // Toggle swtich, verify the alert bar appears and its status
```

Comment:
Prior art, but the 👀 see what the 👀 see.
```suggestion
      // Toggle switch, verify the alert bar appears and its status
```